sktm/__init__.py

```diff
@@ -184,9 +185,12 @@ def check_patchwork(self):
             # Add patchset summaries for all patches staying pending for
             # longer than 12 hours
             patchsets += cpw.get_patchsets(
-                    self.db.get_expired_pending_patches(cpw.baseurl,
-                                                        cpw.projectid, 43200))
-            # For each patchset summary
```

Comment:
Not the comment, the unused `sourceid` below it. As I said at some point before, we can use unnamed variable `_` to avoid both pylint complaints and keeping the function call.